vllm/model_executor/models/baichuan.py

```diff
@@ -385,7 +423,8 @@ def load_weights(self,
                 param = params_dict[name]
                 weight_loader = getattr(param, "weight_loader",
                                         default_weight_loader)
-                weight_loader(param, loaded_weight)
+                weight_loader(param, loaded_weight, weight_owner=weight_owner)
+        torch.cuda.empty_cache()
```

Comment:
Why do we need to manually call the cache release here in each model file (instead of as a part of the util)?

Reply:
Fixed